src/main/java/org/openstreetmap/atlas/geography/atlas/multi/MultiAtlas.java

```diff
@@ -192,12 +192,13 @@ public static MultiAtlas loadFromPackedAtlas(final Iterable<? extends Resource>
         {
             throw new CoreException("Can't create an atlas from zero resources");
         }
-        return new MultiAtlas(
-                Iterables
-                        .translate(resources,
-                                resource -> PackedAtlas.load(resource)
-                                        .subAtlas(filter, AtlasCutType.SOFT_CUT).get()),
-                lotsOfOverlap);
+        final Iterable<Atlas> validAtlases = Iterables
+                .translate(Iterables
+                        .stream(Iterables.translate(resources,
+                                resource -> PackedAtlas.load(resource).subAtlas(filter,
+                                        AtlasCutType.SOFT_CUT)))
+                        .filter(optional -> optional.isPresent()), optional -> optional.get());
```

Comment:
Small nit:
`Optional::isPresent` and `Optional::get`

Reply:
good call!